src/cascadia/TerminalApp/GlobalAppSettings.h

```diff
@@ -72,6 +72,8 @@ class TerminalApp::GlobalAppSettings final
     GETSET_PROPERTY(std::wstring, WordDelimiters); // default value set in constructor
     GETSET_PROPERTY(bool, CopyOnSelect, false);
     GETSET_PROPERTY(bool, CopyFormatting, false);
+    GETSET_PROPERTY(bool, WarnAboutLargePaste, true);
+    GETSET_PROPERTY(bool, WarnAboutMultiLinePaste, true);

```

Comment:
Should these be enabled by default?

Reply:
I think I was initially reluctant to enabled by default for both, but I've sat on it for a bit and I concur, enabled by default seems sensible.